package.json

```diff
@@ -19,8 +19,7 @@
     "test": "jest --detectOpenHandles --runInBand",
     "test:custom-lint-rule": "eslint tools/eslint/rules/valid-constructors.test.ts",
     "test:dev": "jest --detectOpenHandles --runInBand --watch",
-    "prepare": "tsc",
-    "prepack": "cp -r src/bin/commands/template lib/bin/commands/template",
+    "prepare": "mkdir -p lib/bin/commands && cp -r src/bin/commands/template lib/bin/commands/template && tsc",
```

Comment:
Moving the `prepack` steps to `prepare` which is run when `npm link`ing and `npm pack`ing. (`prepare` is the DRYest solution?)

Reply:
Fixed this undoing the changes to the `prepare` command in favour of `npm pack`.